src/main/java/seedu/address/logic/commands/ListImportantCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonHasHighImportancePredicate;
+
+/**
+ * Finds and lists all persons in address book who are of high importance contacts.
+ */
+public class ListImportantCommand extends Command {
+
+    public static final String COMMAND_WORD = "impts";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all person who have been set as a "
+            + "contact with high importance and displays them as a list with index numbers.\n";
```

Comment:
Might want to stay consistent with either `person` or `contact`! 😄

Reply:
Thanks for the feedback! Just pushed the necessary changes 👍